samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/ItemAddListener.java

```diff
@@ -711,6 +711,9 @@ public void saveItem(ItemAuthorBean itemauthor) throws FinFormatException{
      		item = new PublishedItemFacade();
      	}
       }
+
+	  item.setIsExtraCredit(bean.getIsExtraCredit());
```

Comment:
Can you check tabs vs spaces here? In an editor like vim, use "set list" to see tab chars